hack/jenkins/e2e.sh

```diff
@@ -181,231 +214,266 @@ GCE_PARALLEL_FLAKY_TESTS=(
-    : ${PROJECT:="kubernetes-jenkins"}
-    ;;
-
-  # Runs the performance/scalability tests on GCE. A larger cluster is used.
-  kubernetes-e2e-gce-scalability)
-    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-scalability"}
-    : ${E2E_NETWORK:="e2e-scalability"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Performance\ssuite"}
-    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-scalability"}
-    : ${PROJECT:="kubernetes-jenkins"}
-    # Override GCE defaults.
-    MASTER_SIZE="n1-standard-4"
-    MINION_SIZE="n1-standard-2"
-    MINION_DISK_SIZE="50GB"
-    NUM_MINIONS="100"
-    # Reduce logs verbosity
-    TEST_CLUSTER_LOG_LEVEL="--v=1"
-    # Increase resync period to simulate production
-    TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
+    : ${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
+    : ${KUBE_GCE_INSTANCE_PREFIX="e2e-gce-1-1"}
+    : ${KUBE_GCS_STAGING_PATH_SUFFIX:="release-1.1"}
+    : ${PROJECT:="k8s-jkns-e2e-gce-release"}
+    : ${ENABLE_DEPLOYMENTS:=true}
     ;;
 
-  # Runs tests on GCE soak cluster.
-  kubernetes-soak-continuous-e2e-gce)
-    : ${E2E_CLUSTER_NAME:="gce-soak-weekly"}
+  # Runs "disruptive" tests on GCE on the release candidate branch,
+  # sequentially, against the latest 1.1 ci release.
+  # The autoscaling test is not disruptive, however there is no better suite
+  # to run it.
+  kubernetes-e2e-gce-disruptive-1.1)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-disruptive-1.1"}
     : ${E2E_DOWN:="false"}
-    : ${E2E_NETWORK:="gce-soak-weekly"}
-    : ${E2E_UP:="false"}
+    : ${E2E_NETWORK:="gce-e2e-disruptive-1-1"}
     : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
-          ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
-          ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
-          ${GCE_SOAK_CONTINUOUS_SKIP_TESTS[@]:+${GCE_SOAK_CONTINUOUS_SKIP_TESTS[@]}} \
+          ${REBOOT_SKIP_TESTS[@]:+${REBOOT_SKIP_TESTS[@]}}\
+          ) --ginkgo.focus=$(join_regex_no_empty \
+          ${DISRUPTIVE_TESTS[@]:+${DISRUPTIVE_TESTS[@]}} \
+          "Autoscaling\sSuite.*via\sreplicationController" \
+          "GCE\sL7\sLoadBalancer\sController"
           )"}
-    : ${KUBE_GCE_INSTANCE_PREFIX:="gce-soak-weekly"}
+    : ${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
+    : ${KUBE_GCE_INSTANCE_PREFIX="e2e-gce-disruptive-1-1"}
+    : ${KUBE_GCS_STAGING_PATH_SUFFIX:="disruptive-1.1"}
+    # TODO: move into its own project
     : ${PROJECT:="kubernetes-jenkins"}
+    : ${ENABLE_DEPLOYMENTS:=true}
+    : ${ENABLE_DAEMONSETS:=true}
     ;;
 
-  # Runs non-flaky tests on GCE on the release-latest branch,
-  # sequentially. As a reminder, if you need to change the skip list
-  # or flaky test list on the release branch, you'll need to propose a
-  # pull request directly to the release branch itself.
-  kubernetes-e2e-gce-release)
-    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-release"}
-    : ${E2E_DOWN:="false"}
-    : ${E2E_NETWORK:="e2e-gce-release"}
+  kubernetes-e2e-gke-subnet)
```

Comment:
Can you please add a `TODO` & file an issue for figuring out what to do with the GKE & trusty tests?